src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -709,7 +698,7 @@ define(function (require, exports, module) {
                     _doLaunchAfterServerReady(doc);
                 })
                 .fail(function () {
-                    _showWrongDocError();
+                    console.log("Live preview: no document to preview.");
```

Comment:
we dont show any dialogs in phoenix. the live preview page will have a placeholder "nothing to preview" page